genai-perf/genai_perf/profile_data_parser/llm_profile_data_parser.py

```diff
@@ -161,14 +184,47 @@ def _parse_requests(self, requests: dict) -> Metrics:
                     chunked_inter_token_latency.append(round((t2 - t1) / num_token))
                 chunked_inter_token_latencies.append(chunked_inter_token_latency)
 
+                # (per-session) calculate llm metrics
```

Comment:
I think if you are putting comment headers like this, it can be a new method.

Reply:
Not really since we would have to pass bunch of values as arg

Reply:
Fine with me. We can revisit if its a problem another time.